lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -388,6 +388,26 @@ public DecodedJWT verify(DecodedJWT jwt) throws JWTVerificationException {
         return jwt;
     }
 
+    @Override
+    public Boolean isValid(String token) {
+        try {
+            verify(token);
+            return Boolean.TRUE;
+        } catch (Exception e) {
+            return Boolean.FALSE;
+        }
+    }
+
+    @Override
+    public Boolean isValid(DecodedJWT jwt) {
```

Comment:
When would a decoded JWT be used? As in, why would someone pass in an already decoded token here? What would the `verify()` function check?

Reply:
the reason I added the isValid(DecodedJWT jwt) just to keep it consistent with verify methods. As in verify method take either token or DecodedJWT.

Reply:
I may have misunderstood your question. the reason behind to add this method is to 
1) the equivalent method verify throws JWTException which is not very functional programming friendly
2) when Exception is thrown some monitoring tools which does the instrumentation of code to see for any exceptions are showing a lot of JWT exception, when the intent in the code is just to verify whether it is a valid JWT token.